recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.7.1" %}
+{% set md5 = "1e09362a3f35d589f942359441050a6e" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  entry_points:
+    - coverage = coverage.cmdline:main
+    - coverage3 = coverage.cmdline:main
+    - coverage3.5 = coverage.cmdline:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Very minor nit, but could we move these two above `entry_points`. It should help keep this compact and it will be quick to get to the build number if needed.